api/src/app.controller.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
Could you add a brief comment describing the purpose of this and other `ts` files here? I can even just be descriptions from https://docs.nestjs.com/first-steps#setup. Since this is new for all of us, I'd like to err on the side of over-documentation.

Reply:
Done!